lab-teamTruffleShuffle/index.js

```diff
@@ -0,0 +1,122 @@
+'use strict';
+
+//const fs = require('fs');
+const readWrite = require('./lib/read-write');
+const transform = require('./lib/transform');
+
+var fileName = `${__dirname}/./assets/bitmap.bmp`;
+
+newBitmapInverted(fileName);
+newBitmapGrayScale(fileName);
+newBitmapBlackOut(fileName);
+newBitmapRed(fileName);
+newBitmapBlue(fileName);
+newBitmapGreen(fileName);
+
+function newBitmapInverted(fileName, callback) {
```

Comment:
You defined the callback pattern well here, but didn't actually pass a callback into the invocations above. You also didn't utilize the callback interior to this function definition.